src/services/blocks/BlocksTraceService.ts

```diff
@@ -0,0 +1,102 @@
+import { BlockHash } from '@polkadot/types/interfaces';
+import { InternalServerError } from 'http-errors';
+
+import {
+	BlocksTrace,
+	BlocksTraceOperations,
+} from '../../types/responses/BlocksTrace';
+import { AbstractService } from '../AbstractService';
+import { BlockTraceResponse, isBlockTrace, isTraceError, Trace } from './trace';
+
+const DEFAULT_TARGETS = 'pallet,frame,state';
+// :extrinsic_index & frame_system::Account
+const DEFAULT_KEYS =
+	'3a65787472696e7369635f696e646578,26aa394eea5630e07c48ae0c9558cef7b99d880ec681799c0cf30e8886371da9';
+const UNEXPECTED_RPC_RESPONSE = 'Unexpected response to state_traceBlock RPC';
+
+export class BlocksTraceService extends AbstractService {
+	/**
+	 * Get the state traces for a block.
+	 *
+	 * @param hash `BlockHash` to get traces at.
+	 */
+	async traces(hash: BlockHash): Promise<BlocksTrace> {
+		const [{ number }, traceResponse] = await Promise.all([
+			this.api.rpc.chain.getHeader(hash),
+			// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+			// @ts-ignore
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-call
+			this.api.rpc.state.traceBlock(
+				hash,
+				DEFAULT_TARGETS,
```

Comment:
So why is this hardcoded to `DEFAULT_TARGETS` and `DEFAULT_KEYS`? 

Is it possible for users to enter their own log targets and storage keys?